### DISCUSSION TITLE
Examples of using rt-TDDFT in the periodic system

### DISCUSSION THREAD

**Discussion by Satinelamp** [deepmodeling/abacus-develop#1597] [General]
### Background

the rt-TDDFT method is often used in the molecular system, which works well. But in the periodic system for example, when an H2O molecule is absorbed on the surface of TiO2, it could be very tricky to excite electrons so the hot electron could end up in the molecular orbital of H2O. I notice that the examples of rt-TDDFT shipped with ABACUS are about the molecular system, maybe an example about the periodic system would be useful.

### Describe the solution you'd like

An example of using rt-TDDFT in the scenario of an H2O molecule is absorbed on the surface of TiO2

**Comment by mohanchen** [comment]
We are working on the rt-TDDFT

**Comment by NiranjanAgaram** [comment]
Hello,

I was wondering if you could explain to me why I have been cc'd on
this email. Could you please let me know if there is anything specific
that you would like me to contribute or if there is anything I can do
to assist with this matter?

Thank you in advance for your assistance.

Best regards,

Niranjan

**Reply by Otas79** [reply]
Oh yes I want hack Facebook

**Reply by raccoonsteven** [reply]
> Oh yes I want hack Facebook 

Puhleaze.  Facebook hacked into everything in your life.  Go download your data from them and you'll shit yourself on how much they encroached into your life.  They probably know that you secretly collect toe jam and eat it while rocking yourself in the bathroom closet.
